### ISSUE TITLE
Standardize output to obj.value?

### ISSUE THREAD

**Issue by droberson** [gen0cide/gscript#67]
The majority of functions store the output in obj.value, however many functions have unique variable names for their output on a function-by-function basis. Should this be standardized to obj.value when the function only logically returns one thing to make scripting easier and more consistent?

**Comment by droberson** [comment]
errors are also stored with non-consistent names. should this be made uniform as well?

**Comment by gen0cide** [comment]
Originally, we returned a single, variable type, object that you had to determine if an error occurred by what it was. When we moved to the generator to generate the Javascript <=> Golang bridge code for the virtual machine, I wanted to be able to return things like error objects with their messages. 

To translate this paradigm between Go and Javascript, it would need to be supported in both Javascript, as well as the otto runtime's interfaces for bridging native code. Unfortunately both do not. I'm working on a new strategy here for the v1.0.0 release that should alleviate a lot of this pain. Stay tuned.
